Mathlib/Tactic/Linter/DirectoryDependency.lean

```diff
@@ -228,7 +228,6 @@ def allowedImportDirs : NamePrefixRel := .ofArray #[
   (`Mathlib.Logic, `Mathlib.Util),
   (`Mathlib.Logic, `Mathlib.Tactic),
   (`Mathlib.Logic.Fin.Rotate, `Mathlib.Algebra.Group.Fin.Basic),
-  (`Mathlib.Logic.Hydra, `Mathlib.GroupTheory),
```

Comment:
How did this happen? You're touching neither of the two folders

Reply:
This line is literally repeated a few lines below. (I can move this to a separate PR if you prefer.)

Reply:
Ah sure! Just found it surprising in this PR